com.github.textosaurus.json

```diff
@@ -0,0 +1,33 @@
+{
+    "id": "com.github.textosaurus",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "textosaurus",
+    "rename-icon": "textosaurus",
+    "finish-args": [ "--device=dri",
+                     "--share=ipc",
+                     "--share=network",
+                     "--socket=x11",
+                     "--socket=wayland",
+                     "--filesystem=xdg-download",
+                     "--talk-name=org.kde.StatusNotifierWatcher",
+                     "--talk-name=org.freedesktop.Notifications" ],
+    "modules": [
+        {
+            "name": "textosaurus",
+            "buildsystem": "qmake",
+            "post-install": [
+                "install -d /app/share/icons/hicolor/512x512/apps",
+                "install -m644 /app/share/pixmaps/textosaurus.png /app/share/icons/hicolor/512x512/apps/textosaurus.png"
```

Comment:
Pass `-D` here and you can remove the previous line. You will also want to delete the file from `pixmaps` and honestly just change that upstream as nobody should be using that directory.